@xen-orchestra/fs/src/fs.spec.js

```diff
@@ -28,6 +28,8 @@ const unsecureRandomBytes = n => {
 
 const TEST_DATA_LEN = 1024
 const TEST_DATA = unsecureRandomBytes(TEST_DATA_LEN)
+const WRITE_TEST_DATA_LEN = 256
+const WRITE_TEST_DATA = unsecureRandomBytes(WRITE_TEST_DATA_LEN)
```

Comment:
This is not clear that this data should be smaller than `TEST_DATA`.

What if someone change one or the other in the future and breaks the tests?